examples/vdi_profile_example.py

```diff
@@ -0,0 +1,103 @@
+# -*- coding: utf-8 -*-
+"""
+Creating power demand profiles using bdew profiles.
+
+Installation requirements
+-------------------------
+This example requires at least version v0.1.4 of the oemof demandlib. Install
+by:
+    pip install 'demandlib>=0.1.4,<0.2'
+Optional:
+    pip install matplotlib
+
+SPDX-FileCopyrightText: Uwe Krien <krien@uni-bremen.de>
+
+SPDX-License-Identifier: MIT
+
+"""
+
+import datetime
+
+from matplotlib import pyplot as plt
+
+from demandlib import vdi
+
+# The following dictionary has been created by "workalendar"
+# pip3 install workalendar
+# from workalendar.europe import Germany
+# cal = Germany()
+# holidays = dict(cal.holidays(2017))
+
+holidays = {
+    datetime.date(2017, 1, 1): "New year",
+    datetime.date(2017, 4, 14): "Good Friday",
+    datetime.date(2017, 4, 17): "Easter Monday",
+    datetime.date(2017, 5, 1): "Labour Day",
+    datetime.date(2017, 5, 25): "Ascension Thursday",
+    datetime.date(2017, 6, 5): "Whit Monday",
+    datetime.date(2017, 10, 3): "Day of German Unity",
+    datetime.date(2017, 10, 31): "Reformation Day",
+    datetime.date(2017, 12, 25): "Christmas Day",
+    datetime.date(2017, 12, 26): "Second Christmas Day",
+}
+
+
+my_houses = []
+for n in range(1):
+    my_houses.append(
+        {
+            "N_Pers": 3,
+            "name": "EFH_{0}".format(n),
+            "N_WE": 1,
+            "Q_Heiz_a": 6000,
+            "TRY": 4,
+            "copies": 24,
+            "house_type": "EFH",
+            "sigma": 4,
+            "Q_TWW_a": 1500,
+            "W_a": 5250,
```

Comment:
Language: "EFH", "WE", "Heiz" and "TWW" are German, all others should work in English. (First, I misinterpreted WE as weekend.)

Comprehensibility: What does W_a mean?

Reply:
This has bugged me, too. The reason is sticking to the notation in the norm itself. Even though there is an English version of VDI 4655, that one still uses the German notation. To name a few:
Q: Heat
W: Electrical work
TWW: Domestic hot water
TT: Typical day
a: Year
WE: Flats

Therefore ``W_a`` is the electrical work of one year in kWh.

While this may not be ideal, I'd still argue that sticking to the source notation is very important, at least for the internal code. Making the public API English is something that might be worth considering, as a compromise.

Reply:
You are right, it might be better to use English abbreviations in the code.